src/main/scala/dotty.tools.sbtplugin/DottyPlugin.scala

```diff
@@ -94,10 +95,10 @@ object DottyPlugin extends AutoPlugin {
       },
 
       scalaBinaryVersion := {
-        if (isDotty.value)
-          "0.1" // TODO: Fix sbt so that this isn't needed
-        else
-          scalaBinaryVersion.value
+        CrossVersion.partialVersion(scalaVersion.value) match {
+          case Some((0, minor)) => s"0.$minor"
```

Comment:
I'm not sure if this is the right thing here, cc/ @smarter Now that nightly builds are 0.2, we should at least not return the constant "0.1". Should I preserve the TODO comment?